dragula.js

```diff
@@ -62,6 +66,11 @@ function dragula (initialContainers, options) {
     var op = remove ? 'remove' : 'add';
     touchy(documentElement, op, 'mousedown', grab);
     touchy(documentElement, op, 'mouseup', release);
+    if (remove) {
+      document.removeEventListener('mousemove', onMouseMove);
+    } else {
+      document.addEventListener('mousemove', onMouseMove);
```

Comment:
You should replace all of this for `touchy(document, op, 'mousemove', onMouseMove);`